content/docs/command-reference/gc.md

```diff
@@ -5,7 +5,7 @@ Remove unused files and directories from <abbr>cache</abbr> or [remote storage].
 ## Synopsis
 
 ```usage
-usage: dvc gc [-h] [-q | -v] [-w] [-a] [-T] [--all-commits]
+usage: dvc gc [-h] [-q | -v] [-w] [--rev <commit>] [-n <num>] [-a] [-T] [-A]
               [--date <YYYY-MM-DD>] [--all-experiments]
               [-c] [-r <name>] [-f] [-j <number>]
               [-p [<path> [<path> ...]]]
```

Comment:
We need to reorg these. Maybe

```suggestion
usage: dvc gc [-h] [-q | -v]
              [-w] [--rev <commit>] [-n <num>] [-a] [-T] [-A]
              [--date <YYYY-MM-DD>] [--all-experiments]
              [-p [<path> [<path> ...]]]
              [-c] [-r <name>] [-f] [-j <number>]
```

Reply:
Please commit ☝🏼 if you agree @daavoo cc @dberenbaum then we can merge, I think. Thanks